Patches/Dragon's Descent/ThingDefs_Races/Patch_Races_Animal_Common_Dragon.xml

```diff
@@ -0,0 +1,1121 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<!-- Making the common dragons CE compatible -->
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+			<!-- ==== Check for the mod. ==== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>Dragon's Descent</modName>
+			</li>
+
+			<!-- Common dragon base. -->
+			<!-- Giving it a shape. -->
+			<li Class="PatchOperationAddModExtension">
+				<xpath>Defs/ThingDef[@Name="DragonRaceBase"]</xpath>
+				<value>
+					<li Class="CombatExtended.RacePropertiesExtensionCE">
+						<bodyShape>Quadruped</bodyShape>
```

Comment:
I'd consider investigate using `BirdLike` which is used for Thrumbos, it may be a better fit.